Samples/Sample XML/StudentSchoolAttendance.xml

```diff
@@ -20,7 +20,7 @@
 		<SessionReference>
 			<SessionIdentity>
 				<SessionName>2021-2022 Fall Semester</SessionName>
-				<SchoolYear>2021-2022</SchoolYear>
+				<SchoolYear>2021</SchoolYear>
```

Comment:
`SchoolYear` is defined as an integer, but the sample files have long had this string in them! This can be problematic for some API clients. Even the `Ed-Fi-Core.xsd` in this repository defines as an integer:

```none
  <xs:simpleType name="SchoolYearType">
    <xs:annotation>
      <xs:documentation>Identifier for a school year.</xs:documentation>
      <xs:appinfo>
        <ann:TypeGroup>Enumeration</ann:TypeGroup>
      </xs:appinfo>
    </xs:annotation>
    <xs:restriction base="xs:int">
...
```